integration/setup/setup.go

```diff
@@ -1,41 +1,66 @@
 // +build k8srequired
 
-package integration
+package setup
 
 import (
 	"io/ioutil"
 	"log"
 	"os"
+	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
-	"github.com/giantswarm/aws-operator/integration/template"
 	"github.com/giantswarm/e2e-harness/pkg/framework"
 	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/aws-operator/integration/client"
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/aws-operator/integration/teardown"
+	"github.com/giantswarm/aws-operator/integration/template"
 )
 
 const (
 	awsResourceValuesFile = "/tmp/aws-operator-values.yaml"
 )
 
-func installAWSResource() error {
-	awsResourceChartValuesEnv := os.ExpandEnv(template.AWSResourceChartValues)
-	d := []byte(awsResourceChartValuesEnv)
+func HostPeerVPC(c *client.AWS, f *framework.Framework) error {
```

Comment:
Only thing that really changed is the injection here. I would like to make this more object oriented but this is for a later step. I am still discovering how to do things best.